python-threatexchange/threatexchange/cli/tests/dataset_cmd_test.py

```diff
@@ -0,0 +1,65 @@
+from threatexchange.cli.tests.e2e_test_helper import ThreatExchangeCLIE2eTest
+
+
+class DatasetCommandTest(ThreatExchangeCLIE2eTest):
```

Comment:
Thanks for adding this test!
Is there are reason you skipped using the cvs flag in one of them?